client/packages/invoices/src/Prescriptions/DetailView/Toolbar.tsx

```diff
@@ -70,6 +79,12 @@ export const Toolbar: FC = () => {
     });
   };
 
+  const defaultPrescriptionDate =
+    DateUtils.getDateOrNull(prescriptionDate) ??
+    DateUtils.getDateOrNull(pickedDatetime) ??
+    DateUtils.getDateOrNull(createdDatetime) ??
```

Comment:
If it's not been at least picked do we want this to just show today? Even if we created it a few days ago?

Reply:
I don't think I see the case for picked date? Would think this should show created date, unless the prescription is backdated, in which case show that date?

Possibly out of scope, but I almost wonder about showing created date and prescription date separately? At the very least, I don't find it particularly intuitive that the date in the list view and the date in the detail view don't always match (list view always shows created date...)

Reply:
> I don't think I see the case for picked date? Would think this should show created date, unless the prescription is backdated, in which case show that date?
> 
> Possibly out of scope, but I almost wonder about showing created date and prescription date separately? At the very least, I don't find it particularly intuitive that the date in the list view and the date in the detail view don't always match (list view always shows created date...)

What about the case where it was created 2 days ago, then 1 day later you picked your items and verified it. You want the prescription date to show as yesterday when you re-open it right?
That was my logic anyway...

This whole question shows up because we've started showing the date on the prescription, previously you never saw it apart from the list view.

Speaking of which, it's a great point, that only created date is in the ListView. If we show something similar to this for prescription date, it's going to mess with sorting though :(

@mark-prins - and thought on that? Should we create new issue?

Reply:
I think the simple solution is to just add `PickedDate` as another column in the List View...

Reply:
I don't see the need for picked date in the mix. The prescription date is functioning as an adjustable created date - any actions taken against the prescription are a separate concern.
My take: remove picked date from that list of fallbacks, and show prescription date in the list view.
I'd also show 'created date' in the right hand panel - that's an extra bit of information which is interesting to know, when it differs from prescription date

Reply:
My argument is that picked date is the date when the stock movement occurs so should be the one we show as the prescription date? But I agree it adds complexity, will try to see if I can get created_date/backdated_date as prescription date showing in the list view without breaking sorting...

Reply:
@mark-prins @jmbrunskill The original request was to be able to backdate a precription when it's being created and it seems to be most logical to me that it's the date that affects the stock that is entered and displayed in that field. Doesn't really make sense to me to display anything else in there. Being able to backdate a prescription after it's been picked is fine as long as the rules about picking the stock available on that date are followed. So I guess setting it to the current date for a new prescription then only ever the picked date is what makes sense to me. I hope that answers the question you're asking! Probably pointless anyway since it's already merged and I think it agrees with the eventual conclusion 😄

Reply:
Thanks @GaryWilletts you validated my thinking at least!